group_vars/tigerdata/qa.yml

```diff
@@ -22,4 +22,48 @@ mailcatcher_install_location: "/usr/local/lib/ruby/gems/3.1.0/gems/mailcatcher-0
 app_datacite_user: '{{ vault_qa_datacite_user }}'
 app_datacite_password: '{{ vault_qa_datacite_password }}'
 app_datacite_host: 'api.test.datacite.org'
-app_datacite_prefix: '10.80021'
\ No newline at end of file
+app_datacite_prefix: '10.80021'
+
+sidekiq_worker_name: tiger-data-workers
+sidekiq_worker_threads: 4
+redis_bind_interface: '0.0.0.0'
+
+rails_app_vars:
+  - name: APP_SECRET_KEY_BASE
+    value: '{{ app_secret_key }}'
+  - name: APP_DB
+    value: '{{ app_db_name }}'
+  - name: APP_DB_USERNAME
+    value: '{{ app_db_user }}'
+  - name: APP_DB_PASSWORD
+    value: '{{ app_db_password }}'
+  - name: APP_DB_HOST
+    value: '{{ app_db_host }}'
+  - name: APPLICATION_HOST
+    value: '{{ app_host_name }}'
+  - name: APPLICATION_HOST_PROTOCOL
+    value: '{{ application_host_protocol }}'
+  - name: HONEYBADGER_API_KEY
+    value: '{{ vault_honeybadger_api_key }}'
+  - name: MEDIAFLUX_TRANSPORT
+    value: 'https'
+  - name: MEDIAFLUX_PORT
+    value: '{{ vault_mediaflux_port }}'
+  - name: MEDIAFLUX_HOST
+    value: '{{ vault_mediaflux_host }}'
+  - name: MEDIAFLUX_DOMAIN
+    value: '{{ vault_mediaflux_domain }}'
+  - name: MEDIAFLUX_USER
+    value: '{{ vault_mediaflux_user }}'
+  - name: MEDIAFLUX_PASSWORD
+    value: '{{ vault_mediaflux_password }}'
+  - name: DATACITE_USER
+    value: '{{ app_datacite_user }}'
+  - name: DATACITE_PASSWORD
+    value: '{{ app_datacite_password }}'
+  - name: DATACITE_HOST
+    value: '{{ app_datacite_host }}'
+  - name: DATACITE_PREFIX
+    value: '{{ app_datacite_prefix }}'
+  - name: BUNDLE_GEMS__CONTRIBSYS__COM
```

Comment:
Because this is overridden, I assumed that one must override the entire block of `rails_app_vars` key-value tuples.

Reply:
Good catch that PDC Describe isn't set up the same way. I made a ticket to fix that: https://github.com/pulibrary/pdc_describe/issues/1666